### PR TITLE
Merge `WorkflowRun` type stub back to source

### PR DESCRIPTION
**Pull request by trim21**

### CODE REVIEW

**Review by EnricoMi — CHANGES_REQUESTED** [review]
I like defining the attributes with their getter functions, but we should stick to one layout while migrating type stubs back: #2553

**Comment by trim21** [comment]
I found that instance attribute is not necessary

File: github/WorkflowRun.py [hunk 1]
```diff
@@ -216,22 +209,16 @@ def get_artifacts(self) -> "PaginatedList[github.Artifact.Artifact]":
...
     @property
     def rerun_url(self) -> str:
         self._completeIfNotSet(self._rerun_url)
         return self._rerun_url.value
 
-    _workflow_url: Attribute[str]
-
     @property
     def workflow_url(self) -> str:
         """
```
Comment (EnricoMi) [inline github/WorkflowRun.py:224]: You removed these docs from other methods, should remove here then as well.

**Review by EnricoMi — APPROVED** [review]
LGTM, minor change requested.

File: github/WorkflowRun.py [hunk 2]
```diff
@@ -142,79 +156,52 @@ def workflow_id(self):
         return self._workflow_id.value
```
Comment (EnricoMi) [inline github/WorkflowRun.py:156]: Sorry, but this property misses type annotation and lacks doc string removal.

**Review by EnricoMi — CHANGES_REQUESTED** [review]

**Comment by EnricoMi** [comment]
Please request review when done, so this flags up as ready for me. Hard to keep track which PR is ready and which are in progress.

**Comment by trim21** [comment]
> Please request review when done, so this flags up as ready for me. Hard to keep track which PR is ready and which are in progress.

Sorry, I thought this PR is done when I trigger review request. Just didn't notice I missed one property.

File: github/WorkflowRun.py [hunk 3]
```diff
@@ -21,200 +21,184 @@
...
-    def pull_requests(self):
-        """
-        :type: list of :class:`github.PullRequest.PullRequest`
-        """
+    def pull_requests(self) -> list[PullRequest]:
         self._completeIfNotSet(self._pull_requests)
         return self._pull_requests.value
 
     @property
     def created_at(self):
```
Comment (EnricoMi) [inline github/WorkflowRun.py:171]: missing typing

File: github/WorkflowRun.py [hunk 4]
```diff
@@ -21,200 +21,184 @@
...
-        """
-        :type: datetime.datetime
-        """
         self._completeIfNotSet(self._created_at)
         return self._created_at.value
 
     @property
     def updated_at(self):
```
Comment (EnricoMi) [inline github/WorkflowRun.py:176]: missing typing

**Review by EnricoMi — CHANGES_REQUESTED** [review]
Sorry, two more missing, please double-check as well.

**Comment by trim21** [comment]
it's all typed this time!

**Review by EnricoMi — APPROVED** [review]
Looks like it is.

**Pull request merged**